### PR TITLE
Speed-up Travis

### PR DESCRIPTION
**Pull request by meyerbaptiste**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | N/A
| License       | MIT
| Doc PR        | N/A

The coverage now takes 10 minutes instead of 15 minutes before.

### CODE REVIEW

**Comment by dunglas** [comment]
Thanks @meyerbaptiste, good job!

File: tests/Fixtures/app/AppKernel.php [hunk 1]
```diff
@@ -38,6 +38,14 @@ class AppKernel extends Kernel
 {
     use MicroKernelTrait;
 
+    public function __construct(string $environment, bool $debug)
+    {
+        parent::__construct($environment, $debug);
+
+        // patch for behat/symfony2-extension not supporting %env(APP_ENV)%
+        $this->environment = $_SERVER['APP_ENV'] ?? $environment;
```
Comment (sroze) [inline tests/Fixtures/app/AppKernel.php]: why not `getenv('APP_ENV')`?

Comment (dunglas) [reply tests/Fixtures/app/AppKernel.php]: because it's not thread safe, and for consistency with Flex/Symfony.

**Comment by teohhanhui** [comment]
The best way to speed up Travis build is to use: https://docs.travis-ci.com/user/build-stages/

**Comment by meyerbaptiste** [comment]
PR @teohhanhui? 😄

**Comment by teohhanhui** [comment]
@meyerbaptiste Just out of curiosity, how does this PR achieve the speed-up? :smile:

**Comment by meyerbaptiste** [comment]
By fixing the cache of our test app. In fact, the Symfony2 Behat extension can't resolve `env: '%env(APP_ENV)%'`. So the test app recreated a cache directory every time we ran Behat: `app/cache/env_APP_ENV_<a random sequence of numbers>`.

**Pull request merged**